http_service/bugbug_http/utils.py

```diff
@@ -30,3 +36,91 @@ def get_hgmo_stack(branch: str, revision: str) -> list:
     r = requests.get(url)
     r.raise_for_status()
     return r.json()["changesets"]
+
+
+# A simple TTL cache to use with models. Because we expect the number of models
+# in the service to not be very large, simplicity of implementation is
+# preferred to algorithmic efficiency of operations.
+#
+# Called an 'Idle' TTL cache because TTL of items is reset after every get
+Key = TypeVar("Key")
+Value = TypeVar("Value")
+
+
+class IdleTTLCache(Generic[Key, Value]):
```

Comment:
We don't use this anywhere, so better to just keep ReadthroughTTLCache and remove this

Reply:
Ack! Forgot to delete this. Will do!